datasetinsights/__main__.py

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
Why rename file `main.py` to `__main__.py`?

Reply:
This is a fix to the error.
/usr/local/bin/python: can't find '__main__' module in 'datasetinsights'